### DISCUSSION TITLE
Uno.UI.Xaml.Core.VisualTree: Visual Tree was not found.

### DISCUSSION THREAD

**Discussion by WenHong-Lau** [unoplatform/uno#17622] [General]
I'm currently debugging an Android application and encountering numerous "Uno.UI.Xaml.Core.VisualTree: Visual Tree was not found." messages. This issue is causing significant delays in rendering controls, and in some cases, controls are not rendering at all.

In this repro, the controls were rendered but there are still Uno.UI.Xaml.Core.VisualTree: Visual Tree was not found. in the output.
[TestProj.zip](https://github.com/user-attachments/files/16329814/TestProj.zip)

**Comment by MartinZikmund** [comment]
Created an issue #17627
